content/events/22-23/lan.md

```diff
@@ -6,14 +6,34 @@ end: '22-10-2022 03:00'
 location: Therminal, Hoveniersberg 24, 9000 Gent
 description: Zeus' elfde LAN-Party!
 header_text_background: true
-banner: 
+banner: https://pics.zeus.gent/zTVVLv1sQAlbS1WFZG60zQQbk4eOjvWIPudB9Uhd.jpg
 locationlink: $therminal
 ---
 
-# Geeks Attack XI
+# Do you wanna play a game ?
+Het is weer de tijd voor de jaarlijkste LAN party van Zeus WPI. Voor iedereen die tot in de laatste uurtjes gezamelijk wilt gamen.
 
-Op **21 oktober** organiseert Zeus zijn elfde LAN-Party: **Geeks Attack XI**!
+## Info
 
-Details worden later nog aangevuld, aanvangstijd kan wijzigen.
+**Datum**: Vrijdag 21 oktober - Zaterdag 22 oktober
+
+**Tijd**: 18:00 tot wanneer we niet meer kunnen
+
+**Locatie**: Therminal, Hoveniersberg 24, Gent
+
+**Meenemen**: Stekkerdoos + ethernetkabel
```

Comment:
```suggestion
**Meenemen**: Stekkerdoos + ethernetkabel + jouw pc
```

Maybe? Alhoewel er natuurlijk ook couchgaming gaat zijn (mss kunnen we daar zelfs nog iets over zeggen), maar voor de pc games moeten ze hun eigen pc voorzien (al is dat mss wel obvious als je je stekkerdoos en ethernetkabel moet meenemen)

Reply:
Doe pc/laptop?

Reply:
ook goed 👍